geode-core/src/main/java/org/apache/geode/internal/cache/map/RegionMapDestroy.java

```diff
@@ -64,52 +64,61 @@
   private boolean doPart3 = false;
   private boolean retainForConcurrency = false;
 
+  private boolean abortDestroyAndReturnFalse;
+  private RegionEntry oldRegionEntry;
+  private RegionEntry newRegionEntry;
+  private RegionEntry regionEntry;
+  private RegionEntry tombstone;
+  private boolean haveTombstone;
+  private boolean doContinue;
+
   public RegionMapDestroy(InternalRegion internalRegion, FocusedRegionMap focusedRegionMap,
       CacheModificationLock cacheModificationLock) {
     this.internalRegion = internalRegion;
     this.focusedRegionMap = focusedRegionMap;
     this.cacheModificationLock = cacheModificationLock;
   }
 
-  public boolean destroy(EntryEventImpl event, boolean inTokenMode, boolean duringRI,
-      boolean cacheWrite, boolean isEviction, Object expectedOldValue, boolean removeRecoveredEntry)
+  public boolean destroy(final EntryEventImpl eventArg, final boolean inTokenModeArg,
```

Comment:
Why rename everything "*Arg"? My preference would be for the original names of the arguments.